platform/lib/utils/sh.js

```diff
@@ -44,7 +44,9 @@ function sh(commandLine, ...options) {
   const command = fragments[0];
   const args = fragments.splice(1);
 
-  console.log(`$ ${command} ${args.join(' ')}`);
+  if (!opts.quiet) {
+    console.log(`$ ${command} ${args.join(' ')}`);
```

Comment:
would it make sense to use the [util logger](https://github.com/ampproject/amp.dev/blob/future/platform/lib/utils/log.js)?

Reply:
The logger util is used to provide scoped logs. I would keep the `console.log` statements as we don't have a specific scope here